docs/book/src/user/quick-start.md

```diff
@@ -640,6 +640,12 @@ spec:
 {{#/tab }}
 {{#/tabs }}
 
+To verify the control plane is up and running ensure the control plane machine
+`PHASE` is `running`.
+```
+kubectl get machines --selector cluster.x-k8s.io/control-plane=true
+```
```

Comment:
Is the Machine Phase of running sufficient here? I would expect the Machine to be running prior to the control plane being available (bootstrapping has not yet completed)

Reply:
I believe it is the bootstrap controller that is responsible for setting the Machine state to Running when a ProviderID is set and a Kubernetes Node with same ProviderID is in Ready state. 
_Instead of mentioning the phase we could specify if a ProviderID is available._ 

However, if your comment is regarding that the node may be Ready but the control plane API isn't accepting requests yet then we wouldn't be able to verify the control plane is up and running unless we try and target the cluster with the kubeconfig which is the next step anyways.

Reply:
sgtm